wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.swift

```diff
@@ -369,10 +369,10 @@ final class ConversationViewController: UIViewController {
     // MARK: Resolve 1-1 conversations
 
     private func resolveConversationIfOneOnOne() {
-        let isOneOnOneConversation = conversation.conversationType == .oneOnOne
-            || conversation.conversationType == .group && (conversation.localParticipants.count == 2)
-
-        guard isOneOnOneConversation && conversation.messageProtocol == .proteus else {
```

Comment:
If this is a fake one on one, then the conversation type will be computed as `.oneOnOne` by a custom accessor, even if the underlying value is `.group`.